cmake_example/main.c

```diff
@@ -71,19 +74,22 @@ int main( void )
                        exampleTaskStack,
                        &exampleTaskTCB );
 
+    /* Start the scheduler. */
     vTaskStartScheduler();
 
-    /* should never get here. */
     for( ; ; )
     {
+        /* Should not reach here. */
     }
-
-    return 0;
```

Comment:
We should change the return type of main to void if we are removing the return 0.

Reply:
updated.